apps/site/src/lib/components/Navbar/Navbar.tsx

```diff
@@ -75,8 +75,24 @@ function Navbar({ identity }: NavbarProps) {
 					<NavLinkItem href="/">Schedule</NavLinkItem>
 					<NavLinkItem href="/">Resources</NavLinkItem>
 					<NavLinkItem href="/">Stage</NavLinkItem> */}
+					<NavLinkItem
+						href="http://tinyurl.com/ih24-incident-form"
+						target="_blank"
+						rel="noopener noreferrer"
+					>
+						Report Incident
+					</NavLinkItem>
+					<NavLinkItem
+						href="https://irvinehacks-2024.devpost.com/"
```

Comment:
Could we add a redirect in `next.config.js` so that `https://irvinehacks.com/devpost` redirects to this URL?